h/routes.py

```diff
@@ -64,6 +64,7 @@ def includeme(config):
     config.add_route('badge', '/api/badge')
     config.add_route('token', '/api/token')
     config.add_route('api.users', '/api/users')
+    config.add_route('api.profile', '/api/profile')
```

Comment:
Ah, ok: good to know. For the sake of clarity, when the route names and the paths have different sort orders, which one should take precedence? As far as I can tell, all the blocks that are already sorted have naming conventions that mean it’s not possible to tell.